samples/javascript/extensions.csproj

```diff
@@ -0,0 +1,12 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+	<WarningsAsErrors></WarningsAsErrors>
+	<DefaultItemExcludes>**</DefaultItemExcludes>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.webJobs.Extensions.DurableTask" Version="1.4.1" />
```

Comment:
This version of the extension doesn't include the required changes, does it? Should we increment the extension version as part of this PR?

Reply:
Oh! Yes, if we can. That would be great.